Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7.1
+FROM golang:1.7.3
```

Comment:
If the OSXCross stuff is no longer in here, can this (and should this) use alpine now?